src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -150,3 +150,27 @@ def wait_for_namespace(namespace, sleep_duration=10)
         sleep sleep_duration
     end
 end
+
+def wait_for_async_service_operation(service_instance_name, retries=0)
+    service_instance_guid = capture("cf service --guid #{service_instance_name}")
+    return { :success => false, :reason => :not_found } if service_instance_guid == 'FAILED'
+    attempts = 0
+    loop do
+        puts "# Waiting for service instance #{service_instance} to complete operation..."
+        service_instance_info = cf_curl("/v2/service_instances/#{service_instance_guid}")
+        return { :success => true } unless service_instance_info.key?('entity')
```

Comment:
In ruby, people tend to just say `… unless service_instance_info['entity']` as a test (and rely on `nil` being falsy)